rls/src/test/java/io/grpc/rls/CachingRlsLbClientTest.java

```diff
@@ -482,7 +488,12 @@ public ManagedChannel createResolvingOobChannel(String target) {
       } catch (IOException e) {
         throw new RuntimeException("cannot create server: " + target, e);
       }
-      return InProcessChannelBuilder.forName(target).directExecutor().build();
+      return InProcessChannelBuilder.forName(target).directExecutor();
+    }
+
+    @Override
+    public ManagedChannel createResolvingOobChannel(String target) {
```

Comment:
I think your code does not call this method, so no need to override.

Reply:
fixed. PTAL